lib/identity_cache/serializer.rb

```diff
@@ -0,0 +1,83 @@
+module IdentityCache
+  module Serializer # :nodoc:
+    DEHYDRATE_EVENT = "dehydration.identity_cache"
+    HYDRATE_EVENT   = "hydration.identity_cache"
```

Comment:
It looks like we are being inconsistent on naming here.  The records aren't actually being serialized to or deserialized from a sequence of bytes, so this isn't really a serializer.  That's why the terms hydration and dehydration were used, even though they aren't as familiar.

Reply:
Switched to `Encoder`